src/osd/scheduler/mClockScheduler.h

```diff
@@ -171,19 +175,19 @@ class mClockScheduler : public OpScheduler, md_config_obs_t {
     }
   }
 
+  /// Set the bandwidth cost per io for the osd
+  void set_osd_bandwidth_cost_per_io();
+
 public:
   mClockScheduler(CephContext *cct, int whoami, uint32_t num_shards,
     int shard_id, bool is_rotational, MonClient *monc);
   ~mClockScheduler() override;
 
-  // Set the max osd capacity in iops
-  void set_max_osd_capacity();
-
-  // Set the cost per io for the osd
-  void set_osd_mclock_cost_per_io();
+  /// Set the max osd capacity in iops - random write @ 4KiB
```

Comment:
tiny nit:

```suggestion
  /// Set the max osd capacity in iops (at 4 KiB random write)
```

Reply:
The method is removed now and is replaced with `mClockScheduler::set_osd_capacity_params_from_config()`